src/views/v3/Bridge/Routes/index.tsx

```diff
@@ -107,69 +107,56 @@ const Routes = ({
     mobile,
   ]);
 
-  const routesWithQuotes = useMemo(() => {
-    return routesList.filter((rs) => quotes[rs] !== undefined);
-  }, [routesList, quotes]);
+  const routesWithQuotes = routesList.filter((rs) => quotes[rs] !== undefined);
 
-  const fastestRoute = useMemo(() => {
-    return routesWithQuotes.reduce(
-      (fastest, route) => {
-        const quote = quotes[route];
-        if (!quote || !quote.success) return fastest;
+  const fastestRoute = routesWithQuotes.reduce(
```

Comment:
For all these data manipulation functions which have nothing really to do with react, they can be just moved into simple utils.ts type file and exported as functions. It will also help with if you want to unit test them. Embedded here will make it hard to test the logic. + If in the future you want to go back and memoize them, you can still using memoize-one library which is pretty performant.